### PR TITLE
Updated the logging to better align with repo goals, README.md

### PR DESCRIPTION
**Pull request by tysam-code**
Logging is a bit more stable (i.e. no crash bugs when using a normal naked 'print' inside of the main training loop, which seems generally user-hostile). Logging is also now, I believe, more hackable/straightforward and does not require an external library (rich). Unfortunately, this comes at the expense of some long lines and a more spartan printout.

I also updated README.md to be a bit less verbose. That's something I surely love doing.

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -471,13 +482,13 @@ def init_split_parameter_dictionaries(network):
 def main():
     # Initializing constants for the whole run.
     net_ema = None ## Reset any existing network emas, we want to have _something_ to check for existence so we can initialize the EMA right from where the network is during training
-                  ## (as opposed to initializing the network_ema from the randomly-initialized starter network, then forcing it to play catch-up all of a sudden in the last several epochs)
+                   ## (as opposed to initializing the network_ema from the randomly-initialized starter network, then forcing it to play catch-up all of a sudden in the last several epochs)
```
Comment (tysam-code) [inline main.py:485]: eh,,, we'll get it in the next go around i guess. herk.

File: main.py [hunk 2]
```diff
@@ -539,8 +548,8 @@ def main():
 
               # we only take the last-saved accs and losses from train
               if epoch_step % 50 == 0:
-                  accuracy_train = (outputs.detach().argmax(-1) == targets).float().mean().item()
-                  loss_train = loss.cpu().item()/batchsize
+                  train_acc = (outputs.detach().argmax(-1) == targets).float().mean().item()
+                  train_loss = loss.detach().cpu().item()/batchsize
```
Comment (tysam-code) [inline main.py:552]: Not sure how I missed this before, but this could have caused a few issues, especially if I ended up 'cleverly' doing inplace computational stuff on the loss before printout at some point in the future, or did something like appending it to a list that held the graph around even longer.
